MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -45,7 +45,8 @@ Please see [Before Development](MODULE_EVALUATION_TEMPLATE#before-development) f
   * _This is not applicable to libraries_
 * Module descriptor MUST include interface requirements for all consumed APIs (3, 5)
   * _This is not applicable to libraries_
-* Third party dependencies use an Apache 2.0 compatible license (2)
+* Inclusion of third party dependencies complies with [ASF 3rd Party License Policy](https://apache.org/legal/resolved.html) (2)
```

Comment:
So to be clear then, you are suggesting that some new module `mod-foo`'s readme should 

>  identify the third-party product and its licensing, and provide a url to the its homepage

?  I agree that's probably the easiest way for a TC module evaluator to find it.  It does presumably add a burden for the module developers; I'm not sure any of our modules' readme files currently do that.

Other apps seem to have a single licenses UI somewhere that lists all of the third-party OSS dependencies across all included modules.  That pattern would reduce duplication when we include the same dependency across modules.  But it does separate the license acknowledgement from the specific module that uses it, which is maybe wrong.  And it would be a bit harder for the module evaluator to check.

I'm not trying to start a huge issue here, if we don't want to tackle this -- and with all license questions, we may want to involve CC.  My point is that just saying "comply with that Apache policy" is not as easy as it sounds, and possibly not more actionable than what we have now.

Reply:
I guess I just respectfully disagree on this.  The PR tries to resolve _which_ licenses by incorporating the Apache policy, which itself **requires** the reporting for Category B, which is meaningless without a _how_.  So in this situation, I don't see how we can separate the _which_ and _how_.  But I respect that you disagree, and it's worth talking through at TC.  And I agree with the initial intent, the policy would be great to incorporate.

Reply:
No I think a README note is definitely sufficient!  But 
1. The criterion should say so (for Category B licenses at least).
2. Others (devs) may not like the solution and prefer something else, so it's worth a brief discussion.

Reply:
> The README file is already mentioned in https://apache.org/legal/resolved.html . If we repeat this in our module acceptance criteria why don't we also repeat the NOTICE file, LICENSE file and "any attribution/notice requirements in the specific license in question" requirements?

Sure, fair question.  I read `resolved.html` as being clear that users **must** comply with "any attribution/notice", LICENSE and NOTICE.  Where as using README is just a suggestion:

> An appropriate and prominent label is a label the user will read while learning about the distribution - for example in a README,

and so FOLIO would be saying that in our project, the README is the specific "appropriate and prominent label" we have selected.